src/components/daydream-controls.js

```diff
@@ -6,7 +6,7 @@ var checkControllerPresentAndSetup = trackedControlsUtils.checkControllerPresent
 var emitIfAxesChanged = trackedControlsUtils.emitIfAxesChanged;
 var onButtonEvent = trackedControlsUtils.onButtonEvent;
 
-var DAYDREAM_CONTROLLER_MODEL_BASE_URL = 'https://cdn.aframe.io/controllers/google/';
+var DAYDREAM_CONTROLLER_MODEL_BASE_URL = window.AFRAME_CONFIG.ASSET_BASE_URL + 'controllers/google/';
```

Comment:
Why is this necessary?

Reply:
Just updated PR description at the top. Relying on a network connection is undesirable for mobile apps. So, these assets are bundled in the `cordova-plugin-aframe` package and loaded statically from a `file://` URL.

Reply:
Is not fair to assume that most mobile devices will have internet connection? Most apps today don't work offline.

Reply:
It depends. Apple sometimes rejects apps that don't work offline. Or, I might be designing an app specifically made to work offline or just LAN. Same with desktop/Electron. I think there are valid app use cases for loading assets locally rather than hitting a CDN. Same goes for the polyfillConfig DPDB_URL and other data files loaded by A-Frame dependencies. 

If you feel like these changes don't belong in the core, another idea I had was to maintain a branch or fork, basically a custom A-Frame build, that gets bundled with [cordova-plugin-aframe](https://github.com/benallfree/cordova-plugin-aframe) since that's how people will be using the package anyway.

Do you think that would be better than what this PR is asking to do? Even with the changes I'm proposing in this PR, this package will never work in Cordova by itself. It needs other Cordova-specific dependencies like `file://`-based XHR and Cordova's File API.

Reply:
Let’s try to accommodate in A-Frame itself. I think we can converge on something surgical. 

FYI, Those models won’t be loaded on iOS. Maybe we can change those url when need arises?

Reply:
Can Android and iOS devices attach to various external controllers? I'm not familiar with that.

In any case I know `/fonts` does get loaded. That's what motivated the creation of a dynamic base URL (see `src/components/text.js`).

Also, Cordova targets Electron. Those controllers could be available there. 

Thinking in terms of Cordova compatibility, the asset base URL seems like a predictable issue that Cordova users will face.